apps/hash-integration-worker/src/activities.ts

```diff
@@ -1,6 +1,120 @@
-export const createIntegrationActivities = () => ({
-  // eslint-disable-next-line @typescript-eslint/require-await,@typescript-eslint/no-unused-vars
-  async helloWorldActivity(params: {}): Promise<String> {
-    return "Hello, world!";
+import {
+  Attachment,
+  Comment,
+  Connection,
+  CustomView,
+  Cycle,
+  Document,
+  Issue,
+  IssueLabel,
+  LinearClient,
+  LinearDocument,
+  Organization,
+  Project,
+  ProjectMilestone,
+  Team,
+  User,
+} from "@linear/sdk";
+
+const readNodes = async <T>(connection: Connection<T>): Promise<T[]> => {
+  const nodes = connection.nodes;
+  while (connection.pageInfo.hasNextPage) {
+    // eslint-disable-next-line no-param-reassign
+    connection = await connection.fetchNext();
+    nodes.push(...connection.nodes);
+  }
+  return nodes;
+};
+
+export const createLinearIntegrationActivities = (createInfo: {
+  linearClient: LinearClient;
+}) => ({
+  async me(): Promise<User> {
+    return await createInfo.linearClient.viewer;
```

Comment:
I would make the return of all these functions either just `return x` or `return await x`.

The main difference is that `return await` means that this function will show in the stack trace, whereas if you do not await the return here it won't. `return await` also technically adds a tiny bit more work although it will make no practical difference here

You can read a bit more here: https://eslint.org/docs/latest/rules/no-return-await